src/api-v1.js

```diff
@@ -321,6 +321,45 @@ api.declare({
 
 api.declare({
   method: 'get',
+  route: '/worker-type-last-modified/:workerType',
+  name: 'workerTypeLastModified',
+  input: undefined,  // No input
+  output: 'get-worker-type-last-modified.json#',
+  title: 'Get Worker Type Last Modified Time',
+  stability:  base.API.stability.stable,
+  description: [
+    'This method is provided to allow workers to see when they were',
+    'last modified.  The value provided through UserData can be',
+    'compared against this value to see if changes have been made',
+    'If the worker type definition has not been changed, the date',
+    'should be identical as it is the same stored value.',
+    
+  ].join('\n'),
+}, async function (req, res) {
+  let workerType = req.params.workerType;
+
+  let worker;
+  try {
+    worker = await this.WorkerType.load({workerType: workerType});
+
+    // We do this because John made a mistake in the V1->V2
+    // schema update and there was a typo :(
+    let workerjson = worker.json();
+    return res.reply(_.pick(workerjson, 'workerType', 'lastModified'));
```

Comment:
Nope, because that property isn't defined in the worker type definition.  I could acheive what you're suggesting by doing:
```javascript
return res.reply({
  workerType: workerjson.workerType,
  workerModified: workerjson.lastModified,
});
```
but I chose not to because I'd like to make the return of this function a pure subset of the full worker type definition, rather eventually removing the `workerModified` field at some point in the future.

@gregarndt do you know if you're using `UserData.workerModified` in the docker-worker?